src/default_panels/GraphCreatePanel.js

```diff
@@ -57,6 +57,12 @@ const GraphCreatePanel = (props, {localize: _, setPanel}) => {
         }}
         attr="z"
       />
+      <DataSelector
+        label={{
+          '*': _('Measure'),
```

Comment:
I don't really understand that key there: `*`
https://plotly.github.io/schema-viewer/?attribute=measure

could you point me to the reference that explains why its `*`?

Reply:
It's a way to give it different names depending on trace type - it's just a construct in `DataSelector`